dashboard/app/models/concerns/user/purgeable.rb

```diff
@@ -8,6 +8,7 @@ module User::Purgeable
 
   included do
     SYSTEM_DELETED_USERNAME = 'sys_deleted'
+    SOFT_DELETED_USER_TTL = 28.days
```

Comment:
Since it is already under the `User` namespace, it is better to avoid repeating "user" in the constant name
```suggestion
    SOFT_DELETED_TTL = 28.days
```

```suggestion
    SOFT_DELETED_RECORD_TTL = 28.days
```